streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -795,17 +789,20 @@ public Serde valueSerde() {
      * @return an configured instance of value Serde class
      */
     public Serde defaultValueSerde() {
+        Object valueSerdeConfigSetting = null;
         try {
+            valueSerdeConfigSetting = get(VALUE_SERDE_CLASS_CONFIG);
```

Comment:
Why don't we initialize it to `get(VALUE_SERDE_CLASS_CONFIG);` like we did in `defaultKeySerde`?